perfkitbenchmarker/package_managers.py

```diff
@@ -237,6 +239,8 @@ def InstallPackages(self, packages):
 
   def Install(self, package_name):
     """Installs a PerfKit package on the VM."""
+    if self.is_static and not self.install_packages:
```

Comment:
Is these a reason to restrict this to only static VMs? I can see this being useful for even custom images.